mountpoint-s3/src/main.rs

```diff
@@ -143,7 +143,6 @@ struct CliArgs {
         long,
         help = "Desired throughput in Gbps",
         value_name = "N",
-        default_value = "10",
```

Comment:
remove this default value give it is specified by :https://github.com/awslabs/aws-c-s3/blob/d7bfe602d6925948f1fff95784e3613cca6a3900/source/s3_client.c#L398-L402

Reply:
do you still want to remove this? I think we should.

Reply:
add "[default: 10]" to the `help` since we're removing the actual default value

Reply:
we should not set default customer at cli args. Maximize throughput if customer throughput is missing,